tools/enumerate-devices/rs-enumerate-devices.cpp

```diff
@@ -233,33 +258,35 @@ int main(int argc, char** argv) try
                 << endl;
         }
 
-        if (show_options || show_calibration_data)
-            cout << "\n\nNote:  \"-s\" option is not compatible with the other flags specified,"
-                 << " all the additional options are skipped" << endl;
+        if (show_options || show_calibration_data || show_modes)
+            cout << "\n\nNote:  \"-s\"\\\"-S\" option is not compatible with the other flags specified,"
```

Comment:
Usually / is used for "or", not a backspace \

Reply:
done